automation/taskcluster/lib/tasks.py

```diff
@@ -144,33 +179,52 @@ def craft_beetmover_task(
                 bucket_name = 'maven-production'
                 bucket_public_url = 'https://maven.mozilla.org/'
 
-        version = version.lstrip('v')
         payload = {
+            "artifactMap": [{
+                "locale": "en-US",
+                "paths": {
+                    artifact['taskcluster_path']: {
+                        'checksums_path': '',  # TODO beetmover marks this as required even though it's not needed here
+                        'destinations': [artifact['maven_destination']]
+                    } for artifact in (build_artifacts)
+                },
+                "taskId": build_task_id,
+            }, {
+                "locale": "en-US",
+                "paths": {
+                    artifact['taskcluster_path']: {
+                        'checksums_path': '',  # TODO beetmover marks this as required even though it's not needed here
+                        'destinations': [artifact['maven_destination']]
+                    } for artifact in (sign_artifacts)
+                },
+                "taskId": sign_task_id,
+            }],
             "maxRunTime": 600,
             "upstreamArtifacts": [{
-                'paths': [artifact],
+                'paths': [artifact['taskcluster_path'] for artifact in build_artifacts],
                 'taskId': build_task_id,
                 'taskType': 'build',
-                'zipExtract': True,
+            }, {
+                'paths': [artifact['taskcluster_path'] for artifact in sign_artifacts],
+                'taskId': sign_task_id,
+                'taskType': 'signing',
             }],
             "releaseProperties": {
                 "appName": "snapshot_components" if is_snapshot else "components",
             },
-            "version": "{}-SNAPSHOT".format(version) if is_snapshot else version,
```

Comment:
Just realized this might break our snapshots. I'll check against staging snapshots to ensure this still works.

Reply:
This indeed broke snapshots as that's still expected. Working on a fix.